src/components/Details/Details.js

```diff
@@ -297,6 +304,29 @@ const Details = ({
               tab={tab}
             />
             <TabsSlider tabsList={detailsMenu} initialTab={params.tab} />
+            {params.tab === DETAILS_METRICS_TAB && (
+              <div className="item-header__custom-filters">
+                <MetricsSelector
+                  name="metrics"
+                  metrics={detailsStore.metricsOptions.all}
+                  onSelect={metrics =>
+                    setSelectedMetricsOptions({ endpointUid: selectedItem.metadata.uid, metrics })
+                  }
+                  preselectedMetrics={detailsStore.metricsOptions.preselected}
+                />
+                <DatePicker
+                  className="details-date-picker"
+                  date={detailsStore.dates.value[0]}
+                  dateTo={detailsStore.dates.value[1]}
+                  selectedOptionId={detailsStore.dates.selectedOptionId}
+                  label=""
+                  onChange={handleChangeDates}
+                  type="date-range-time"
+                  timeFrameLimit={TIME_FRAME_LIMITS.MONTH}
+                  withLabels
+                />
+              </div>
+            )}
```

Comment:
`Details` component is the common one. We can't have condition and code related to the specific tab. You need to move this logic out of here.